index.js

```diff
@@ -8,3 +8,8 @@ module.exports = {
 	mapping: require('./lib/mapping'),
 	msearch: require('./lib/msearch')
 };
+
+// doppler auto secret rotation requires a suffix of _ACCESS_KEY_ID, but signed-es-aws-fetch module does not have that in its list of accepted key names
+if (process.env.ES_AWS_ACCESS_KEY_ID) {
```

Comment:
Shouldn't this be better at the beginning of the file?

Reply:
it won't affect them as they aren't required until signedFetch is called, but I'm moving to top as it is more typical to have the exports at the end